demo/transient/transient_snes_mpfaof90.cfg

```diff
@@ -16,10 +16,10 @@ pressure = 1.0e-12 relative
 input_arguments=-max_steps 10 -tdy_regression_test -tdy_regression_test_num_cells_per_process 2 -tdy_regression_test_filename transient-snes-mpfaof90 -pc_type lu
 
 [transient-snes-mpfaof90-dirichlet]
-input_arguments=-max_steps 48 -ic_value 90325 -nx 5 -ny 5 -nz 5 -tdy_water_density exponential -tdy_mpfao_gmatrix_method MPFAO_GMATRIX_TPF -tdy_mpfao_boundary_condition_type MPFAO_DIRICHLET_BC -tdy_regression_test -tdy_regression_test_num_cells_per_process 2 -tdy_regression_test_filename transient-snes-mpfaof90-dirichlet -pc_type lu
+input_arguments=-max_steps 48 -ic_value 90325 -nx 5 -ny 5 -nz 5 -tdy_water_density exponential -tdy_mpfao_gmatrix_method MPFAO_GMATRIX_TPF -tdy_mpfao_boundary_condition_type MPFAO_DIRICHLET_BC -tdy_regression_test -tdy_regression_test_num_cells_per_process 2 -tdy_regression_test_filename transient-snes-mpfaof90-dirichlet -pc_type lu -tdy_output_cell_geo_attributes cell_geo_attributes.bin
 
 [transient-snes-mpfaof90-seepage]
-input_arguments=-max_steps 48 -ic_value 90325 -nx 5 -ny 5 -nz 5 -tdy_water_density exponential -tdy_mpfao_gmatrix_method MPFAO_GMATRIX_TPF -tdy_mpfao_boundary_condition_type MPFAO_SEEPAGE_BC -tdy_regression_test -tdy_regression_test_num_cells_per_process 2 -tdy_regression_test_filename transient-snes-mpfaof90-seepage -pc_type lu
+input_arguments=-max_steps 48 -ic_value 90325 -nx 5 -ny 5 -nz 5 -tdy_water_density exponential -tdy_mpfao_gmatrix_method MPFAO_GMATRIX_TPF -tdy_mpfao_boundary_condition_type MPFAO_SEEPAGE_BC -tdy_regression_test -tdy_regression_test_num_cells_per_process 2 -tdy_regression_test_filename transient-snes-mpfaof90-seepage -pc_type lu -tdy_read_cell_geo_attributes cell_geo_attributes.bin
```

Comment:
@jeff-cohere I have modified these two tests for testing the two new options. The first test outputs a file that is read by the second test. If the first test fails, the second test will automatically fail. Would adding `cell_geo_attributes.bin` in the repo be a better approach?

Reply:
I think the approach you've taken here is fine. At some point in the future, we may want to switch to a more sophisticate testing setup to allow ourselves to express this kind of inter-test dependency.